clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -2429,6 +2435,15 @@ void Generic_GCC::GCCInstallationDetector::AddDefaultGCCPrefixes(
   static const char *const RISCV64Triples[] = {"riscv64-unknown-linux-gnu",
                                                "riscv64-unknown-elf"};
 
+  static const char *const RISCV32beLibDirs[] = {"/lib32", "/lib"};
```

Comment:
Do we have tests for the little endian directories? If so, can we make a big endian test?